packages/DropdownMenu/src/Item.styles.js

```diff
@@ -5,12 +5,17 @@ import stylers from "@paprika/stylers";
 const itemStyles = css`
   ${stylers.fontSize()};
   ${stylers.lineHeight()}
+  color: ${tokens.color.black};
```

Comment:
I know when I was consuming the dropdownMenu component in results and used the non external links I had application styles causing styling issues, so I was experimenting with adding 

```
border-radius: 0;
border-width: inherit;
min-height: inherit;
text-align: left;
``` 
 to fix some of those application styles from affecting this. But can revisit that later.